contrib/internal/httptrace/httptrace.go

```diff
@@ -0,0 +1,62 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package httptrace provides functionalities to trace HTTP requests that are commonly required and used across
+// contrib/** integrations.
+package httptrace
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// StartRequestSpan starts an HTTP request span with the standard list of HTTP request span tags. URL query parameters
```

Comment:
As @katiehockman has also suggested, we should clarify what the "standard list" is or means, specifically. Yes, there is a risk of losing sync between the docs and the code, but perhaps if you make a bulleted list it will be harder to miss.

Reply:
On it yes 👍
We actually can't move the HTTP tags out of `ddtrace/ext/tags.go` since it would be a breaking API change. So I'll just add their list in there for now. But the list might be very long once you support all the tags so I still think some `const ()` block documentation would be better.

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/1286/commits/e989bc6a2b14146ed4a2186b599b1aed413c7699